cli/admin/README.md

```diff
@@ -126,5 +126,69 @@ Note that if you are using the standalone deployment, Bookkeeper commands (and o
 available. For this reason, we encourage you to go a step further and deploy a full Pravega cluster to 
 explore the whole functionality of Pravega Admin CLI.
 
+## Pravega CLI on Kubernetes
+
+The Pravega Admin CLI needs to be executed from inside the Kubernetes cluster, as it requires to reach the services that form a Pravega cluster.
+
+To this end, you can either start a new pod in the cluster and build the tool, or log into a Pravega server pod that already has the Pravega Admin CLI built in. In this section, we will assume the latter option and describe how to use the Pravega Admin CLI from within existing Pravega server pods.
+
+You can access a Segment Store pod in the following way:
+````
+kubectl exec pravega-pravega-segment-store-0 -it bash
+````
+
+Once in the pod, you can run the Pravega Admin CLI:
+```
+./bin/pravega-admin
+    OpenJDK 64-Bit Server VM warning: Option MaxRAMFraction was deprecated in version 10.0 and will likely be removed in a future release.
+    Pravega Admin CLI.
+    
+    Exception reading input properties file: null
+    Initial configuration:
+        pravegaservice.container.count=4
+        bookkeeper.ledger.path=/pravega/pravega/bookkeeper/ledgers
+        cli.security.auth.enable=false
+        cli.security.auth.credentials.password=1111_aaaa
+        pravegaservice.cluster.name=pravega/pravega
+        cli.store.metadata.backend=segmentstore
+        cli.controller.rest.uri=http://localhost:9091
+        cli.security.auth.credentials.username=admin
+        pravegaservice.zk.connect.uri=localhost:2181
+        cli.controller.grpc.uri=tcp://localhost:9090
+```
+
+The initial configuration needs to be modified according to the information of your cluster by accessing the properties file (`./conf/admin-cli.properties`) or using `config set` command.
+For example, this is the configuration we used on a new Pravega cluster deployed via Helm chart as described in the [Pravega Operator documentation](https://github.com/pravega/pravega-operator/tree/master/charts/pravega#installing-the-chart):
+```
+config set cli.controller.rest.uri=pravega-pravega-controller.default:10080
+
+config list
+    pravegaservice.container.count=4
+    bookkeeper.ledger.path=/pravega/pravega/bookkeeper/ledgers
+    cli.security.auth.enable=false
+    cli.security.auth.credentials.password=1111_aaaa
+    pravegaservice.cluster.name=pravega/pravega
+    cli.store.metadata.backend=segmentstore
+    cli.controller.rest.uri=pravega-pravega-controller.default:10080
+    cli.security.auth.credentials.username=admin
+    pravegaservice.zk.connect.uri=localhost:2181
+    cli.controller.grpc.uri=tcp://localhost:9090
```

Comment:
This is also wrong, `localhost` should be replaced with `pravega-pravega-controller.default`.